modules/preview-react/menu/lib/Menu.tsx

```diff
@@ -181,11 +181,12 @@ export default class Menu extends React.Component<MenuProps, MenuState> {
     const children = React.Children.toArray(this.props.children);
     let nextSelectedIndex = 0;
     let isShortcut = false;
-    const itemCount = children.filter(child => {
+    const interactiveItems = children.filter(child => {
       return !(child as React.ReactElement<MenuItemProps>)?.props?.isHeader;
-    }).length;
+    });
+    const interactiveItemCount = interactiveItems.length;
     const firstItem = 0;
-    const lastItem = itemCount - 1;
+    const lastItem = interactiveItemCount - 1;
```

Comment:
While we're at it, we should clarify what this refers to, batch with suggestion on line 215.

```suggestion
    const lastItemIndex = interactiveItemCount - 1;
```